### DISCUSSION TITLE
Deployment with FastAPI

### DISCUSSION THREAD

**Discussion by Ethan-Barr** [vercel/community#2709] [Help] [answered]
### Summary

When I am trying to deploy my FastAPI application to Vercel from the github clone i am getting this error for some reason.

<html>
<body>

WARN! Due to `builds` existing in your configuration file, the Build and Development Settings defined in your Project Settings will not apply. Learn More: https://vercel.link/unused-build-settings
--
15:50:29.135 | > Installing Builder: @now/python
15:50:31.713 | WARN! npm WARN deprecated @now/python@1.2.3: "@now/python" is deprecated and will stop receiving updates January 2021. Please use "@vercel/python" instead.
15:50:31.779 | Installing required dependencies...
15:50:36.535 | Build Completed in /vercel/output [5s]
15:50:36.682 | Deploying outputs...
15:50:38.918 | An unexpected error happened when running this build. We have been notified of the problem. This may be a transient error. If the problem persists, please contact Vercel Support https://vercel.com/help
15:50:40.957 | unexpected_failure: An unexpected error happened when running this build. We have been notified of the problem. This may be a transient error. If the problem persists, please contact Vercel Support https://vercel.com/help

</body>
</html>

### Steps to Reproduce
The base of my FastAPI system with parts taken out

```py
from fastapi.middleware.cors import CORSMiddleware
from fastapi.responses import JSONResponse
from fastapi import FastAPI, requests, Request
import uvicorn



# Variables
app = FastAPI()


# Setup
app.add_middleware(
    CORSMiddleware,
    allow_origins=["http://localhost:3000"],
    allow_credentials=True,
    allow_methods=["*"],
    allow_headers=["*"],
)


# fastapi/Setup --> /api/setup/...

@app.get("/")
def api():
    return "API is running"


# running
if __name__ == '__main__':
    uvicorn.run(app, port=5001, host="0.0.0.0")```

**Comment by amyegan** [comment, marked as answer]
Hi @Ethan-Barr. Please upgrade from `@now/python` to `@vercel/python` and let me know if that fixes the error for on deployments 🙏

**Reply by Ethan-Barr** [reply]
Hey,
I have allready done that and has followed an older post but I am now getting a 500 error upon deployment
![image](https://github.com/vercel/community/assets/70385874/c0919ebf-f90b-4d86-8309-00d131a2835a)

**Reply by amyegan** [reply]
You should be able to get more info from the [Runtime Logs](https://vercel.com/docs/concepts/observability/runtime-logs). Do you see any additional error details there?

**Reply by Ethan-Barr** [reply]
Hey, Dont worry i have fixed it now but thanks anyway 😄
